lib/Sema/CSClosure.cpp

```diff
@@ -96,6 +96,29 @@ class TypeVariableRefFinder : public ASTWalker {
       if (!var)
         return {true, expr};
 
+      if (auto *wrappedVar = var->getOriginalWrappedProperty()) {
+        auto outermostWrapperAttr =
+            wrappedVar->getAttachedPropertyWrappers().back();
```

Comment:
Hmm, `getAttachedPropertyWrappers().front()` should give the outermost wrapper attribute

Reply:
Indeed, I thought that it was in reverse, going to add a method to make it harder to get wrong...

Reply:
Fixed!